dev/breeze/src/airflow_breeze/templates/pyproject_TEMPLATE.toml.jinja2

```diff
@@ -41,7 +41,7 @@
 # `pyproject_TEMPLATE.toml.jinja2` IN the `dev/breeze/src/airflow_breeze/templates` DIRECTORY
 #
 [build-system]
-requires = ["flit_core >=3.2,<4"]
+requires = ["flit_core ==3.10.0"]
```

Comment:
We'll have to add upgrade check as a follow-up to bump this version when flit releases it.